packages/core/src/payment/strategies/paypal-commerce/paypal-commerce-sdk.ts

```diff
@@ -9,6 +9,7 @@ export interface ClickActions {
     resolve(): Promise<void>;
 }
 
+// Moved to package
```

Comment:
What does this comment mean?

Reply:
This comment means that the interface already exist in paypal commerce package and can be removed in the nearest feature. The comments were left to understand which interfaces didn't exist in paypal commerce package

Reply:
I will remove this comments, because the file should be clear and understandable for each developer, not only for me

Reply:
Sounds good, maybe good to add a more meaningful commit. Also we have an issue right now if we move types that need to be exported as part of SDK they need to be duplicated in `core` or else they are not exported correctly as part of SDK build.

Reply:
As far as I understand we can remove that types with all other code that we have in core package after moving all other code to its own package. Right?